health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/Hypothesis.js

```
@@ -191,111 +198,135 @@ const Hypothesis = ({ category, assumptions: initialAssumptions,setShowToast,all
+            heading={t("CONFIRM_NEW_ASSUMPTION")}
+            equalWidthButtons={true}
+            children={
+              category === "CAMPAIGN_VEHICLES"
+                ? [
+                    <LabelFieldPair className="new-assumption-pop">
+                      <span className="bold">{t(`CHOOSE_VEHICLE`)}</span>
+                      <Dropdown
+                        variant="select-dropdown"
+                        t={t}
+                        isMandatory={false}
+                        option={[...state?.VehicleDetails]}
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)